### PR TITLE
Fixed some of the previously skipped tests by adding mocks

### PR DESCRIPTION
**Pull request by lorem--ipsum**
Added a bunch of test utilities who goes by the `*.mock.tsx` extension. While they usually contain, well, mocks (fake data, fake config, etc.) they can also contain logic (see `body-portal.mock.tsx`) meant to be invoked directly from the tests.

### CODE REVIEW

File: run-coverage [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 # Use a reporter that does not take a ton of height
-node node_modules/.bin/istanbul cover _mocha -- --reporter nyan build/common/**/*.mocha.js
```
Comment (vogievetsky) [inline run-coverage]: wtf man, you do not like nyan :-p ? you have a cat!

Comment (lorem--ipsum) [inline run-coverage]: mrow

File: src/client/components/body-portal/body-portal.mock.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    beforeEach(() => {
+      sinon.stub(BodyPortal.prototype, 'render', function() { return this.props.children; });
+      sinon.stub(BodyPortal.prototype, 'teleport');
+    });
+
+    afterEach(() => {
+      (BodyPortal.prototype.render as Sinon.SinonStub).restore();
+      (BodyPortal.prototype.teleport as Sinon.SinonStub).restore();
+    });
+  }
```
Comment (vogievetsky) [inline src/client/components/body-portal/body-portal.mock.tsx]: very nice.

Comment (lorem--ipsum) [inline src/client/components/body-portal/body-portal.mock.tsx]: Yeah, although what I had mixed feelings about is that it's not super easy to tell from the outside that the `disabledBodyPortal` actually handles the enabling part as well... So there's room for improvement, I guess.

**Comment by vogievetsky** [comment]
gr 🎱

**Pull request merged**